src/test/utils.ts

```diff
@@ -323,10 +324,14 @@ export async function execCLI(
       if (currentInputTimeout) {
         clearTimeout(currentInputTimeout);
       }
-      reject(err.toString());
+      if (rejectOnErr) {
+        reject(err.toString());
+      }
     });
 
-    childProcess.on('error', (err: Error) => reject(err));
+    childProcess.on('error', (err: Error) => {
+      if (rejectOnErr) reject(err);
```

Comment:
except this, this not looks good, i missed that

Reply:
Yep, I missed that too. I was playing with errors in child process and forget to discard my changes.